### PR TITLE
Wizard: Add remove user button in users step (HMS-5378)

### PR DESCRIPTION
**Pull request by mgold1234**
this commit add remove user button in users step
Screenshot 2025-01-24 at 17 38 21

JIRA: [HMS-5378](https://issues.redhat.com/browse/HMS-5378)

### CODE REVIEW

File: src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx [hunk 1]
```diff
@@ -115,6 +130,34 @@ const UserInfo = () => {
...
+              className="pf-v5-u-pl-sm pf-v5-u-pt-0 pf-v5-u-pb-0 pf-v5-u-pr-0"
+            >
+              <HelpIcon />
+            </Button>
+          </Popover>
+          <Button variant="secondary" onClick={onRemoveUserClick}>
+            Remove a user
+          </Button>
+        </EmptyStateFooter>
+      </EmptyState>
```
Comment (regexowl) [inline src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx:160]: Nice! I think a remove button will be a good addition.

Let's make it simpler - we already have an empty user state and there's not really a reason to add another one. I'd also drop the popover, "Remove user" as a button text is self-explanatory enough.

Comment (regexowl) [reply src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx:160]: How about something like one of these?

image

The first one is a "plain" button. The second is tertiary with an icon and a tooltip.

**Review by regexowl — CHANGES_REQUESTED** [review]
Can you also add a test to cover the button please? Maybe checking if the empty state with add button reappears when the remove button gets clicked or something like that?

File: src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx (hunk 1 above)
Comment (mgold1234) [reply src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx]: I implement the button with the tooltip. if I remove the emptyState, the rectangle of the button looks wired

File: src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx [hunk 2]
```diff
@@ -62,6 +62,15 @@ const clickAddUser = async () => {
   expect(addUser).toBeEnabled();
   await waitFor(() => user.click(addUser));
 };
+
+
```
Comment (lucasgarfield) [inline src/test/Components/CreateImageWizard/steps/Users/Users.test.tsx:66]: The extra line here is causing the dev-check to fail.

**Review by lucasgarfield — CHANGES_REQUESTED** [review]
Looking good, I like it! Couple of small change requests. Also, would you mind updating your commit message so it has proper spelling and capitalization?

File: src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx [hunk 3]
```diff
@@ -115,6 +120,16 @@ const UserInfo = () => {
           name="user Administrator"
         />
       </FormGroup>
+      <Tooltip position="top-start" content={<div>Remove user</div>}>
```
[the remark this answers is not in the document]
Comment (lucasgarfield) [reply src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx:123]: Ah, I see that the Patternfly example uses them. I'm still not convinced we need them.

[the remark this answers is not in the document]
Comment (mgold1234) [reply src/Components/CreateImageWizard/steps/Users/component/UserInfo.tsx]: right, intersting why they add it in Patternfly

**Review by lucasgarfield — APPROVED** [review]
Working nicely, thank you!
